06_puppy/madhavansu/types.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+// Puppy defines the attributes corresponding to puppy
+type Puppy struct {
+	id    uint
+	breed string
+	color string
+	value float32
```

Comment:
Since `value` is presumably an amount of money, using a `float32` is not a good idea. Here's one article on that, but there are plenty more: https://dzone.com/articles/never-use-float-and-double-for-monetary-calculatio

Reply:
Updated.